jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/Annotator.java

```diff
@@ -73,8 +73,10 @@
      *            JSON property
      * @param propertyName
      *            the name of the JSON property that this getter gets
+     * @param propertyNode
+     *            the schema node defining this property
      */
-    void propertyGetter(JMethod getter, String propertyName);
+    void propertyGetter(JMethod getter, String propertyName, JsonNode propertyNode);
```

Comment:
@ctrimble if we overload `propertyGetter`, then in the rule classes, do we call the new version / the old version / both?